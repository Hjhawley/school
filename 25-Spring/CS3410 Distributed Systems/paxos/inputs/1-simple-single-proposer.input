initialize 3 nodes

// node 3 starts a proposal
at 1001 send prepare request from 3

// deliver all three prepare requests, each triggering a response
at 1002 deliver prepare request message to 2 from time 1001
at 1003 deliver prepare request message to 3 from time 1001
at 1004 deliver prepare request message to 1 from time 1001
