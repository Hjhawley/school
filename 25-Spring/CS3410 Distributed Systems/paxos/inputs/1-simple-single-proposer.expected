--> initialized 3 nodes
--> sent prepare requests to all nodes from 3 with sequence 5003
--> prepare request from 3 sequence 5003 accepted by 2 with no value
--> prepare request from 3 sequence 5003 accepted by 3 with no value
--> prepare request from 3 sequence 5003 accepted by 1 with no value
